static/app/stores/organizationEnvironmentsStore.tsx

```diff
@@ -89,6 +93,6 @@ const storeConfig: StoreDefinition &
 
 const OrganizationEnvironmentsStore = createStore(
   makeSafeRefluxStore(storeConfig)
-) as Store & OrganizationEnvironmentsStoreInterface;
+) as SafeRefluxStore & OrganizationEnvironmentsStoreInterface;
```

Comment:
I noticed we were inconsistent with some of our store types, sometimes we type it as Store other times as SafeStore. Why is that?

Reply:
Not all stores are created with the safeRefluxStore, only the ones with the listenTo in their init methods

Reply:
Got it!